Comments out some optional dependencies
Remove impossibly strict version requirements.
Also remove gmsh as we have no package info, but it still exists.

Author: micketeer@gmail.com
--- requirements.txt.orig	2022-05-31 22:41:21.831456310 +0200
+++ requirements.txt	2022-05-31 22:42:50.275191692 +0200
@@ -1,31 +1,31 @@
 # If you're changing this, consider relaunching:
 # https://gricad-gitlab.univ-grenoble-alpes.fr/ttk/docker-ttk/-/pipelines
 
-numpy==1.19.5 # Python >=3.6
+numpy # Python >=3.6
 
-scipy==1.5.4 # Python >=3.6
+scipy # Python >=3.6
 
-scikit-image==0.17.2 # Python >=3.6
+scikit-image # Python >=3.6
 
-tifffile==2020.8.25 # Python >=3.6
+tifffile # Python >=3.6
 
-matplotlib==3.3.4 # Python >=3.6
+matplotlib # Python >=3.6
 
-SimpleITK==2.0.2
+SimpleITK
 
-progressbar==2.5
+progressbar
 
-python-decouple==3.4
+python-decouple
 
-gmsh==4.8.0
-meshio==4.3.11 # Python >=3.6
-h5py==3.1.0 # Python >=3.6
+#gmsh
+meshio # Python >=3.6
+h5py # Python >=3.6
 
 # Anything later than this causes an abort at the end of sphinx 3.5.2 build
-pybind11==2.5.0 # Python !=3.0, !=3.1, !=3.2, !=3.3, !=3.4, >=2.7
+pybind11 # Python !=3.0, !=3.1, !=3.2, !=3.3, !=3.4, >=2.7
 
 # multigrid solver for random walker segmentation
-pyamg==4.0.0
+pyamg
 
 ############
 # Optional #
@@ -39,11 +39,11 @@
 #ipython==7.16.1 # Python >=3.6
 
 # R (correlated random field)
-rpy2==3.2.2
+#rpy2==3.2.2
 
 #############
 # Graphical #
 #############
-PyQt5==5.14.2 # Python >=3.6
+PyQt5 # Python >=3.6
 #PyQt5-sip==12.8.1 # Python >=3.5
-qimage2ndarray==1.8.3
+qimage2ndarray
